experiments/experiments.go

```diff
@@ -30,14 +42,39 @@ var (
 		NormalisedUploadPaths:      {},
 	}
 
+	Promoted = map[string]string{
+		GitMirrors: fmt.Sprintf("The %s experiment has been promoted to a stable feature. You can safely remove the `--experiment %s` flag without affecting anything", GitMirrors, GitMirrors),
```

Comment:
the `without affecting anything` wording is a bit clunky here, would appreciate pointers on how to england good

Reply:
nah, i like it 💪